src/main/java/com/elastic/support/monitoring/MonitoringExportService.java

```diff
@@ -4,10 +4,8 @@
 import com.elastic.support.diagnostics.DiagnosticException;
 import com.elastic.support.diagnostics.commands.CheckElasticsearchVersion;
 import com.elastic.support.rest.*;
-import com.elastic.support.util.JsonYamlUtils;
-import com.elastic.support.util.ResourceCache;
-import com.elastic.support.util.SystemProperties;
-import com.elastic.support.util.SystemUtils;
+import com.elastic.support.util.*;
```

Comment:
Not a fan of `*` imports, but it's fine for this since the code already has others.

Reply:
Done.

It's weird, IntelliJ keeps adding them back automatically.

Reply:
IntelliJ has pretty low defaults for doing that. Most teams I work on suggest setting the limit to 9999.